src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -100,14 +128,18 @@ class CommonCodes(Enum):
 
         self.print_step(4, "Send ChangeToMode command with NewMode set to %d" % (old_current_mode))
 
-        ret = await self.send_change_to_mode_cmd(newMode=old_current_mode)
+        ret = await self.send_clean_change_to_mode_cmd(newMode=old_current_mode)
         asserts.assert_true(ret.status == CommonCodes.SUCCESS.value, "Changing the mode to the current mode should be a no-op")
 
         if self.check_pics("RVCCLEANM.S.M.CAN_TEST_MODE_FAILURE"):
             asserts.assert_true(self.mode_fail in modes,
                                 "The MODE_CHANGE_FAIL PIXIT value (%d) is not a supported mode" % (self.mode_fail))
             self.print_step(5, "Manually put the device in a state from which it will FAIL to transition to mode %d" % (self.mode_fail))
-            input("Press Enter when done.\n")
+            if self.is_ci:
+                print("Changing mode to Cleaning")
+                await self.send_run_change_to_mode_cmd(1)
+            else:
+                input("Press Enter when done.\n")
```

Comment:
I think this is probably going to cause issues when trying to run in the TH. It doesn't mean it's wrong - it's not. But the TH doesn't currently pipe stuff through to the running scripts. So I think we should get this in, and it may need to be run outside the TH for now. Are you in the cert tool development slack? It's worth starting a conversation there about how the TH can support this test to make it easier for folks to cert.

Reply:
This is tracked [here](https://github.com/project-chip/certification-tool/issues/163).